core/src/main/scala/cats/data/IndexedStateT.scala

```diff
@@ -483,7 +483,7 @@ sealed abstract private[data] class IndexedStateTContravariantMonoidal[F[_], S]
   implicit def G: Applicative[F]
 
   override def unit: IndexedStateT[F, S, S, Unit] =
```

Comment:
nit: can we change this to `override val unit` so we don't have to allocate all those internal things repeatedly (applyF, G.pure, F.trivial).

Reply:
changed